devel/pkgconfig/Portfile

```diff
@@ -38,21 +38,11 @@ set docdir          ${prefix}/share/doc/${name}
 
 # This port is used by clang-3.4 to bootstrap libcxx
 subport ${name}-bootstrap {
-    set myprefix ${prefix}/libexec/libcxx-bootstrap
```

Comment:
Similar to xz, all use of `myprefix` should be removed.

Reply:
done